__tests__/MyComponentAccessibility.test.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { axe, toHaveNoViolations } from '@axe-core/react';
+
+// Adding the toHaveNoViolations matcher
+expect.extend(toHaveNoViolations);
+
+// Test component
+const App = () => (
+  <div>
+    <h1>Hello, world!</h1>
+    <p>This is a sample component.</p>
+  </div>
+);
+
+// Mounting the component and running accessibility checks
+ReactDOM.render(<App />, document.getElementById('root'), async () => {
+  const results = await axe(document.body);
+  expect(results).toHaveNoViolations();

```

Comment:
It’s nice to see an example of a test.  Eventually we’d want to add axe testing to our existing components.  But we have some that I know will fail if we do!